apps/frontend/src/app/login.tsx

```diff
@@ -0,0 +1,15 @@
+import { trpc } from '../utils/trpc';
+
+export const Login = () => {
+  const login = trpc.user.login.useMutation();
+
+  return (
+      <div>
+        <p>TRPC Connected</p>
+
+        <button onClick={() => login.mutate({ email: 'test2@test.com', password: 'testtesttest' })}>
+          Login
+        </button>
+    </div>
+  );
```

Comment:
We may want to consider using something like Ionic to make our lives ez

If not Ionic, then some other component library

Reply:
definitely!